jbpm-task-assigning/kie-server-client-task-assigning/src/main/java/org/kie/server/client/TaskAssigningRuntimeClient.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.client;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.kie.server.api.model.taskassigning.ExecutePlanningResult;
+import org.kie.server.api.model.taskassigning.PlanningItemList;
+import org.kie.server.api.model.taskassigning.TaskDataList;
+import org.kie.server.api.model.taskassigning.TaskInputVariablesReadMode;
+
+public interface TaskAssigningRuntimeClient {
+
+    ExecutePlanningResult executePlanning(PlanningItemList planningItemList, String userId);
```

Comment:
Have you considered using the new SolverManager API? It was designed for use cases such as yours, and I think it would save you a lot of code.

Reply:
SolverManager API wasn't available/delivered at the time this development started and don't know if it's production ready right now. So it wasn't included as part of this development, and I don't see this as a priority right know. When this this version is stable, etc, I'll take a look and evaluate the benefits/drawbacks of adding it.

Reply:
Guys, as I mentioned before, it took too long to setup this initial "running" version, and priority now is to have this version closed, ideally in this sprint. After this, improvements ad-dings are welcome, but will be incorporated in separate JIRAs, I want to have more control on this.

The following JIRA will track this adding

https://issues.redhat.com/browse/PLANNER-1805

Thanks.